guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -41,11 +41,11 @@ ifdef::katello,satellite,orcharhino[]
 ----
 # {package-install} postgresql-contrib
 ----
-.. Connect to the Pulp database:
```

Comment:
It's hard to argue what's right here. You could also argue the service is Pulp and the database is happened to named `pulpcore`. Users can technically change the database name

Reply:
@ekohl so in context of this topic (Postgres external DB), it would make sense to call it Pulp DB, right?

Reply:
Yes, sorry for the late reply. I thin that makes sense.